src/components/TipInput.vue

```diff
@@ -121,6 +121,25 @@ export default {
         };
       },
     }),
+    largestFtTipAmount() {
+      return this.tipUrlStats.tokenTotalAmount.length
+        ? this.tipUrlStats.tokenTotalAmount.reduce(
+          (a, b) => (a.amount > b.amount ? a : b),
```

Comment:
@stanislav-slavov Shouldn't `a.amount` and `b.amount` be shifted by the decimals value of the corresponding token as it was before?